control-station/src/views/Dynamics/Dynamics.tsx

```diff
@@ -0,0 +1,28 @@
+import SensorBox from "@/components/SensorBoxes/Sensors/SensorBox";
+import PodContext from "@/services/PodContext";
+import FrontPod from "@/data/images/FrontPod.png";
+import SidePod from "@/data/images/SidePod.png";
+import "./Dynamics.css";
+import { useContext } from "react";
+function Dynamics() {
```

Comment:
Chore: the code is a little cramped here, let's make sure to have a new line before the function definition and between import groups.